lists-service/config.ini

```diff
@@ -0,0 +1,44 @@
+[DEFAULT]
+PRODUCT_NAME = lists
+GITHUB_REPO_NAME = species-lists
+GITHUB_OWNER = AtlasOfLivingAustralia
+REGION = ap-southeast-2
+# the bootstrap stack names. These are used by the codepipeline stacks to import
+# the codepipeline roles, buckets and codestar connections
+BOOTSTRAP_STACK_NAME = ala-bedrock-cicd-bootstrap-production
+BUCKETS_STACK_NAME = ala-bedrock-cicd-buckets-production
+VPC_STACK_NAME = ala-bedrock-vpc-production
+
+[development]
+# code pipeline
+CODESTAR_CONNECTION = arn:aws:codestar-connections:ap-southeast-2:748909248546:connection/e336fd41-54c2-42e1-97c9-cbd6cc09fe88
+ARTIFACTS_BUCKET = ala-code-pipeline-artifacts-748909-ap-southeast-2-production
+CLOUDFORMATION_SERVICE_ROLE = arn:aws:iam::748909248546:role/cloud-formation-service-role-production
+CODEBUILD_SERVICE_ROLE = arn:aws:iam::748909248546:role/service-role/code-build-service-role-production
+CODEPIPELINE_SERVICE_ROLE = arn:aws:iam::748909248546:role/code-pipeline-service-role-production
+LISTS_ECR_REPOSITORY = 748909248546.dkr.ecr.ap-southeast-2.amazonaws.com/ala-lists
+EKS_CLUSTER_NAME = testing
```

Comment:
All these values are exported from bedrock, regolith or the base component and can be removed. same for the other environments